examples/typescript/lnd/src/nodeHandler.ts

```diff
@@ -0,0 +1,45 @@
+import { LnRpc, Peer, WalletUnlockerRpc } from '@radar/lnrpc';
+
+export default class NodeHandler {
+  constructor(private lnrpc: LnRpc & WalletUnlockerRpc) {}
+
+  getChoices = () => Object.keys(NodeHandler.answers);
+
+  listPeers = async () => {
+    const info = await this.lnrpc.listPeers();
+    return info.peers;
+  };
+
+  seePeerCount = async () => {
+    const info = await this.lnrpc.getInfo();
+    return info.numPeers;
+  };
+
+  handlers = {
+    'See number of peers': this.seePeerCount,
+    'List peers': this.listPeers,
```

Comment:
Open to having better actions here if that matters, I just picked these because they were easy to call. but maybe more intricate actions would be better

Reply:
It would be good to list and open channels. These are a common use case for most apps building on LN. I'm not sure if this would be too complicated to add.